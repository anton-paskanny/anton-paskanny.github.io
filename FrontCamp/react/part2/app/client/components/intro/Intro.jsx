import React from 'react';

import './styles.css';


export default (props) => (
  <div className="intro">
    <h1 className="intro__title">
      Simple blogs app
    </h1>
    <p className="intro__text">
      This is the home page
    </p>
  </div>
)
