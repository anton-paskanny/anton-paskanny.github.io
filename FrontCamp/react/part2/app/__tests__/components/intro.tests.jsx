import React from 'react';
import ReactDOM from 'react-dom';
import { shallow, configure, mount } from 'enzyme';
import renderer from 'react-test-renderer';
import Adapter from 'enzyme-adapter-react-16';

import Intro from '../../client/components/intro/Intro';

configure({ adapter: new Adapter() });

describe('Intro component', () => {
  test('should render Intro component', () => {
    const component = shallow(<Intro />);
    expect(component).toHaveLength(1);
  });

  test('should render correctly', () => {
    const component = renderer.create(<Intro />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
