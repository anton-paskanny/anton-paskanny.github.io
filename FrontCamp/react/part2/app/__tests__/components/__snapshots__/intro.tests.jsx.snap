// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Intro component should render correctly 1`] = `
<div
  className="intro"
>
  <h1
    className="intro__title"
  >
    Simple blogs app
  </h1>
  <p
    className="intro__text"
  >
    This is the home page
  </p>
</div>
`;
